starmark-integration/src/storage/CloudflareEnvAdapter.ts

```
@@ -0,0 +1,134 @@
+    if (this.isCloudflareEnvironment(context)) {
+      return context!.env!;
+    }
+    return process.env;
+  }
+
+  /**
+   * Get a specific environment variable by key
+   */
+  getEnv(key: string, context?: CloudflareContext | null): string | undefined {
+    if (this.isCloudflareEnvironment(context)) {
+      return context!.env![key];
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)